crates/enarx-config/src/lib.rs

```diff
@@ -45,30 +55,52 @@ impl<'de> Deserialize<'de> for FileName {
         let name = String::deserialize(deserializer)?;
 
         if name.contains(':') {
-            return Err(D::Error::custom("invalid name"));
+            return Err(D::Error::custom("invalid value for `name` contains ':'"));
```

Comment:
It looks like `contains ':'` should be the cause for the outermost error.
Normally I'd expect the caller to do something like:
```rust
let name = v.deserialize(de).context("invalid file name")?
````

Reply:
How about
```suggestion
            return Err(D::Error::custom("`name` contains ':'"));
```